internal/backend/remote-state/azure/go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/manicminer/hamilton-autorest v0.2.0 // indirect
 	github.com/mattn/go-colorable v0.1.13 // indirect
 	github.com/mattn/go-isatty v0.0.20 // indirect
+	github.com/mitchellh/colorstring v0.0.0-20190213212951-d06e56a500db // indirect
```

Comment:
This is something I'm not 100% sure on, but it seems to me that change was needed since I added the dependency to `configs` package, which is somehow a dependency of `backend` package. Without this change I was getting failures in "Code Consistency Checks" test on CI. This change and it's related changes are the result of running `make syncdeps`.

Reply:
Took some edits to how the `Detail` of diagnostics are handled in the `format` package, but was able to remove the direct colorization in the `configs` package that subsequently removes the need for these changes.

Included here https://github.com/hashicorp/terraform/pull/34943/commits/7b3e8a1aa8ad16d87f57ffb9ac9b8f136e2046b3

Otherwise dropped the previous commit that included the `go.mod` and `go.sum` changes